client/client.js

```diff
@@ -405,3 +405,20 @@ function expressMetricsEndpointDetail(modelOrUri, windowStartTime, callback) {
   );
 }
 Client.prototype.expressMetricsEndpointDetail = expressMetricsEndpointDetail;
+
+function executorCreate(driver, callback) {
```

Comment:
If the first parameter is an object instead of a string, then the general prototype of the method won't have to change if you add parameters.